Resources/Locale/en-US/accent/german.ftl

```diff
@@ -0,0 +1,221 @@
+accent-german-words-1 = yes
+accent-german-words-1-2 = yea
+accent-german-words-replace-1 = ja
+
+accent-german-words-2 = no
+accent-german-words-replace-2 = nein
+
+accent-german-words-3 = the
+accent-german-words-replace-3 = ze
+
+accent-german-words-4 = shit
+accent-german-words-replace-4 = scheisse
+
+accent-german-words-5 = hospital
+accent-german-words-replace-5 = krankenhaus
+
+accent-german-words-6 = sausage
+accent-german-words-replace-6 = wurst
+
+accent-german-words-7 = sausages
+accent-german-words-replace-7 = wieners
```

Comment:
wiener -> würstchen

Reply:
That's a mouthful to read in English, I'd rather remove the plural alltogether.